src/test/java/org/apache/commons/io/FileSystemTest.java

```diff
@@ -82,6 +85,14 @@ public void testSorted() {
                 assertTrue(chars[i] < chars[i + 1], fs.name());
             }
         }
+
+        for (final FileSystem fs : FileSystem.values()) {
```

Comment:
Have added this for consistency; however it is already implicitly checked that the array is sorted by the methods `testIsLegalName` and `testIsReservedFileName` because they would fail if the array was not sorted.